### PR TITLE
[FIX] Unable to switch to channel in which a thread was previously selected - iPad

### PR DESCRIPTION
**Pull request by reinaldonetof**
## Proposed changes

Switch to channel in which a thread was previously selected

## Issue(s)	

It is not possible to switch to a channel in which a thread was previously selected.
Closes https://github.com/RocketChat/Rocket.Chat.ReactNative/issues/3193

## How to test or reproduce

- Select a thread in Channel A
- Switch to Channel B
- Try to switch back to Channel A

### Before
https://user-images.githubusercontent.com/47038980/125011748-6b23c580-e03f-11eb-9e57-8d63090cf61b.mov

### After
https://user-images.githubusercontent.com/47038980/125011771-72e36a00-e03f-11eb-8935-3257cb93171b.mov

## Types of changes

- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves a current function)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update (if none of the other choices apply)

## Checklist

- [x] I have read the [CONTRIBUTING](https://github.com/RocketChat/Rocket.Chat/blob/develop/.github/CONTRIBUTING.md#contributing-to-rocketchat) doc
- [x] I have signed the [CLA](https://cla-assistant.io/RocketChat/Rocket.Chat.ReactNative)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if applicable)
- [ ] I have added necessary documentation (if applicable)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by gerzonc — APPROVED** [review]
LGTM

File: app/views/RoomsListView/index.js [hunk 1]
```diff
@@ -726,8 +726,7 @@ class RoomsListView extends React.Component {
 	goRoom = ({ item, isMasterDetail }) => {
 		logEvent(events.RL_GO_ROOM);
 		const { item: currentItem } = this.state;
-		const { rooms } = this.props;
-		if (currentItem?.rid === item.rid || rooms?.includes(item.rid)) {
+		if (currentItem?.rid === item.rid) {
```
Comment (diegolmello) [inline app/views/RoomsListView/index.js:729]: We're removing `rooms` reducer from here, but is it being used anywhere?
Is it needed still?

@reinaldonetof

Comment (reinaldonetof) [reply app/views/RoomsListView/index.js:729]: the `rooms` from props are being used at shouldComponentUpdate and componentDidUpdate

**Comment by diegolmello** [comment]
@reinaldonetof Is this still valid?

**Comment by ankar84** [comment]
@gerzonc @diegolmello @GleidsonDaniel hey!
Is there a plans to land that fix to next release?
Thanks!

**Comment by ankar84** [comment]
And also I guess PR https://github.com/RocketChat/Rocket.Chat.ReactNative/pull/4038 fix same issue

**Comment by ankar84** [comment]
@gerzonc @diegolmello @GleidsonDaniel Hello, Team!
Any updates about that fix?

**Pull request closed without merging**